setup.py

```diff
@@ -66,7 +76,7 @@
 builtins._ASTROPY_PACKAGE_NAME_ = PACKAGENAME
 
 # VERSION should be PEP440 compatible (http://www.python.org/dev/peps/pep-0440)
-VERSION = metadata.get('version', '0.1.dev')
+VERSION = metadata.get('version', '0.0.dev0')
```

Comment:
change 0.0 to the next version? Last release was 0.3, so it should be I presume next one will be 0.4?

Reply:
Good catch!

Reply:
I do believe that updating in `setup.cfg` and running e.g. `python setup.py develop` will auto-populate the version information in the required places.

Reply:
ahh, ok, this is new infrastructure then that I haven't been caught up yet ;)

Reply:
If you can, go ahead and set the version to `0.3.1` and regenerate. If not, I can do a new PR with the version changes.